Cargo.toml

```diff
@@ -116,3 +116,7 @@ crate-type = ["bin"]
 [[example]]
 name = "callback"
 crate-type = ["bin"]
+
+[patch.crates-io]
+libc = { git = "https://github.com/rust-lang/libc.git" }
```

Comment:
Why is this required?

Reply:
Without this line, packages depending on libc tries to use the released version of libc.
This line https://github.com/wasmerio/wasmer/pull/1292/files#diff-1cb0a48640bca2c4affbcf2f5e9a81c9R20 wasn't enough

Anyway I'll delete this line after